changelogs/fragments/10227-pacemaker-cluster-and-resource-enhancement.yml

```
@@ -0,0 +1,7 @@
+deprecated_features:
+  - pacemaker_cluster - the parameter ``state`` will become a required parameter in community.general 12.0.0 (https://github.com/ansible-collections/community.general/pull/10227).
+
+minor_changes:
+  - pacemaker_cluster - add ``state=maintenance`` for managing pacemaker maintenance mode (https://github.com/ansible-collections/community.general/pull/10227).
+  - pacemaker_cluster - rename ``node`` to ``name`` and add ``node`` alias (https://github.com/ansible-collections/community.general/pull/10227).
+  - pacemaker_resource - enhance module by removing duplicative code and adding additional unit test (https://github.com/ansible-collections/community.general/pull/10227).
```

Tests are usually not mentioned in the changelog. (They are not user-facing.)
```suggestion
  - pacemaker_resource - enhance module by removing duplicative code (https://github.com/ansible-collections/community.general/pull/10227).
```